docs/en/compute-edition/30/admin-guide/waas/waas-intro.adoc

```diff
@@ -27,6 +27,8 @@ The Inline Defenders evaluate client requests against security policies before r
 
 image::./CNAF-architecture.png[width=650]
 
+*The WAAS serverless offering does not provide the full range of defenses available in other environments. Specifically, it does not support protection against bot attacks or denial of service (DoS) attacks.
```

Comment:
@Kamesh-PaloAlto instead, consider "The WAAS serverless offering does not support protection against bot attacks or denial of service (DoS) attacks."

Also, mention which features in Prisma Cloud provide protection against those attacks.